src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design</id>
+    <version>1.2.0</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Command line tool to precompile Razor views for a project to improve startup time.</description>
+    <tags>dotnet,razor,views</tags>
+    <serviceable>true</serviceable>
+  </metadata>
+  <files>
+    <file src="artifacts/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.dll" target="build/" />
```

Comment:
What does the structure of the package look like? Also, does this work for `net451` targeting apps? Might also need the crossTargeting target (https://github.com/aspnet/RazorTooling/blob/nimullen/msbuild.93/src/Microsoft.AspNetCore.Razor.Tools/buildCrossTargeting/Microsoft.AspNetCore.Razor.Tools.targets) to make this work for multi-targeting projects.